tests/test_tests.py

```diff
@@ -41,6 +41,18 @@ def test_typechecks(self, env):
             {{ none is none }}
             {{ 42 is number }}
             {{ 42 is string }}
+            {{ 42 is integer }}
```

Comment:
Same as the comment below, let's split this out into `pytest.mark.parametrize`.

Reply:
Agreed. Fixed.